webpack.config.prod.js

```diff
@@ -29,17 +30,13 @@ module.exports = {
       {
         test: /\.(jpe?g|png|gif)$/i,
         use: {
-          loader: 'url-loader?limit=10000!img?progressive=true&-minimize',
-          options: {
-            name: '[name].[ext]',
-            outputPath: 'img/'
```

Comment:
`url-loader` and `svg-url-loader` fallback to `file-loader` when a file exceeds the limit set by their respective options. `outputPath` is strictly a `file-loader` option but it will only receive this option when it's set as a query param. With the old configuration, `file-loader` would simply not work. 

Relying on `file-loader` instead of the build script to copy the files that are not inlined results in a cleaner dist directory that contains only the assets referenced in the output files.